src/main/java/org/folio/rest/core/RestClient.java

```diff
@@ -64,6 +64,21 @@ public <T> Future<T> post(String endpoint, T entity, Class<T> responseType, Requ
       .onFailure(log::error);
   }
 
+  public <T> Future<Void> postEmptyResponse(String endpoint, T entity, RequestContext requestContext) {
+    log.info(REQUEST_MESSAGE_LOG_INFO, HttpMethod.POST, endpoint);
+    if (log.isDebugEnabled()) {
```

Comment:
i believe log.isDebugEnabled is not useful in any places
https://stackoverflow.com/questions/17850614/is-log4j-isdebugenabled-necessary-before-using-logger-debug

Reply:
We should not just remove the `if` because then we would map and encode the entity for nothing in most cases. But I guess we can use lambdas instead. I changed the other occurrences in the class for consistency.